docs/applications_showcase.mdx

```diff
@@ -11,10 +11,16 @@ This file contains links to applications that integrate MindsDB. Please append l
   Link: https://ritwickrajmakhal.github.io/ytinsights
 </Card>
 
-<Card title="App Name" icon="link" href="https://github.com/mindsdb/mindsdb">
-  Author: your_name
+<Card title="Condense" icon="link" href="https://github.com/adeyinkaezra123/condense">
+  Author: Ezra Adeyinka
 
-  Link: link_1, link_2, ...
+  Link: https://condense-mindsdb.vercel.app/
+</Card>
+
+<Card title="NpmForecast" icon="link" href="https://github.com/adeyinkaezra123/npmforecast">
+  Author: Ezra Adeyinka
+
+  Link: https://npmforecast.streamlit.app/
```

Comment:
Let's remove this untill it works. We released new version please check

Reply:
Everything is functioning as expected, and I've introduced additional features to predict download frequencies for multiple packages. I appreciate your patience and the support you've provided.

![image](https://github.com/mindsdb/mindsdb/assets/65364356/556f92a1-25cb-4b95-933d-7d8673c89607)

![image](https://github.com/mindsdb/mindsdb/assets/65364356/17aa0009-5652-4233-bb9d-667d175a79d5)

Reply:
Do you mind me writing an article on this? I believe this aligns perfectly with a timeseries prediction use case

Reply:
Yes, please